creator-node/src/serviceRegistry.js

```diff
@@ -316,7 +316,7 @@ class ServiceRegistry {
     this.findReplicaSetUpdatesQueue = findReplicaSetUpdatesQueue
     this.cNodeEndpointToSpIdMapQueue = cNodeEndpointToSpIdMapQueue
     this.manualSyncQueue = manualSyncQueue
-    this.recurringSyncQueue = recurringSyncQueue
+    this.recurringSyncQueue = null
```

Comment:
just disabling recurring sync to test on mad dog to make sure manual sync is picking these up